kite-data/kite-data-core/src/test/java/org/kitesdk/data/spi/filesystem/TestFileSystemView.java

```diff
@@ -251,6 +251,147 @@ public void run() {
 
   @Test
   @SuppressWarnings("unchecked")
+  public void testMoveToTrash() throws Exception {
```

Comment:
Is there a practical way to test items are actually moved to the expected trash folder? Perhaps just checking if the presence of directories under .Trash? It looks to me like these tests validate items are removed but not that they actually are in trash.

If this isn't practical in a unit test we can probably just rely on user testing since these at least verify things are removed as expected.